lib/buildPublicData.js

```diff
@@ -144,6 +209,10 @@ async function main() {
     const overlayData = await Promise.all(overlays.map(o => o.dataType === 'csv' && fetchAndParse(o.dataSource)));
     const parsedOverlays = await handleOverlays(overlays, overlayData);
     const cleanedVariables = handleVariables(variables, categories);
+    const cmsPostsDir = path.join(__dirname, '..', 'public', 'content', 'posts');
+    const cmsStickersDir = path.join(__dirname, '..', 'public', 'content', 'stickers');
+    const cmsPosts = zipMarkdownDirectoryToJsonArray(cmsPostsDir, (post) => injectTaglineIntoPost(post));
+    const cmsStickers = zipJsonDirectoryToArray(cmsStickersDir);
```

Comment:
I've come up with a pattern here that may generally work for arbitrary Markdown/JSON collections in PagesCMS

`cmsStickers` is created by parsing a directory of JSON files created by a PagesCMS collection
`cmsPosts` is created by parsing a directory of Markdown files created by a PagesCMS collection

Both of these optionally offer a predicate that lets you inject additional fields for specific overrides :+1: